components/brex/common.app.mjs

```diff
@@ -0,0 +1,145 @@
+import axios from "axios";
+import { v4 as uuidv4 } from "uuid";
+
+export default {
+  propDefinitions: {
+    postedAtStart: {
+      type: "string",
+      label: "Posted At Start",
+      description: "Shows only transactions with a posted_at_date on or after this date-time. This parameter is the date-time notation as defined by [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6). Example: `2022-12-12T23:59:59.999`",
+      optional: true,
+    },
+    max: {
+      type: "integer",
+      label: "Max",
+      description: "The maximum amount of registered that will be fetched. Defaults to `500`.",
```

Comment:
This says `max` defaults to 500, but in the components, `DEFAULT_LIMIT` is set to 100.
```suggestion
      description: "The maximum amount of registered that will be fetched. Defaults to `100`.",
```